x-pack/plugin/esql/qa/testFixtures/src/main/resources/spatial_shapes.csv-spec

```diff
@@ -0,0 +1,128 @@
+###############################################
+# Tests for GEO_SHAPE type
+#
+
+convertFromString#[skip:-8.12.99, reason:spatial type geography only added in 8.13]
+row wkt = "POLYGON ((30 10, 40 40, 20 40, 10 20, 30 10))"
+| eval pt = to_geography(wkt);
+
+wkt:keyword                                     |pt:geography
+"POLYGON ((30 10, 40 40, 20 40, 10 20, 30 10))" | POLYGON ((30.0 10.0, 40.0 40.0, 20.0 40.0, 10.0 20.0, 30.0 10.0))
+;
+
+convertFromStringArray#[skip:-8.12.99, reason:spatial type geography only added in 8.13]
+row wkt = ["POINT(42.97109630194 14.7552534413725)", "POINT(75.8092915005895 22.727749187571)"]
+| eval pt = to_geography(wkt);
+
+wkt:keyword                                                                           |pt:geography
+["POINT(42.97109630194 14.7552534413725)", "POINT(75.8092915005895 22.727749187571)"] |[POINT(42.97109630194 14.7552534413725), POINT(75.8092915005895 22.727749187571)]
+;
+
+# need to work out how to upload WKT
+#simpleLoad#[skip:-8.12.99, reason:spatial type geography  only added in 8.13]
```

Comment:
We could load the airports file here (but with header changed to geo_shape), and then add another test with polygons later.